docs/src/main/paradox/routing-dsl/directives/method-directives/head.md

```diff
@@ -18,8 +18,8 @@ method `HEAD` are passed on to the inner route. All others are rejected with a
 by the default @ref[RejectionHandler](../../rejections.md#the-rejectionhandler).
 
 @@@ note
-By default, akka-http handles HEAD-requests transparently by dispatching a GET-request to the handler and
-stripping of the result body. See the `akka.http.server.transparent-head-requests` setting for how to disable
+By default, akka-http handles HEAD-requests does not transparently dispatch a GET-request to the handler and
```

Comment:
Thanks for catching this! The sentence doesn't quite flow now, though... I guess we could fix it with something like:

```suggestion
By default, akka-http does not transparently handle HEAD-requests by dispatching a GET-request to the handler and
```

But it might be nicer to restructure even further.

Reply:
I was not really happy with my sentence, I'm gonna to change it and think of a better structure.

Thank you